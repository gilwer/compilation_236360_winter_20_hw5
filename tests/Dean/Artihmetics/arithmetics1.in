void main()
{
	int a1 = 4 + 2*3;
	printi(a1); 
	int a11 = 2*3+4;
	printi(a11);
	int a2 = (3*2-2);
	int a22 = (2-3*2);
	printi(a2);
	printi(a22);
	int a3 = 14-23*1+9*1+15/15+2*0;
	printi(a3);
	int a4 = 56/4+4;
	printi(a4);
	int a5 = (4*(4)+4);
	printi(a5);
	int a6 = 64/2/2;
	printi(a6);
}