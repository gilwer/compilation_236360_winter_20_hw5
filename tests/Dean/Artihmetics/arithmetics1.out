10
10
4
-4
1
18
20
16
